### DISCUSSION TITLE
Version compatibility for pgbouncer

### DISCUSSION THREAD

**Discussion by KaishM** [pgbouncer/pgbouncer#622] [Q&A] [answered]
There is no mention of Pre-Reqs or minimum requirements for each version of pgbouncer at http://www.pgbouncer.org/changelog.html
Is there any documentation somewhere else? 
It would be super helpful to include each version's compatibility such as pgbouncer 1.15 is applicable to Postgres 11.x onwards and Ubuntu 20.0

**Comment by petere** [comment, marked as answer]
PgBouncer is a PostgreSQL protocol proxy, it does not care about which specific version of PostgreSQL you use. (It does not even have to be PostgreSQL, it could be something else that uses the same protocol.)

**Reply by KaishM** [reply]
thanks @petere
